src/test/scala/mesosphere/marathon/core/matcher/base/util/ActorOfferMatcherTest.scala

```diff
@@ -1,46 +1,18 @@
 package mesosphere.marathon
 package core.matcher.base.util
 
-import java.util.UUID
-
 import akka.actor.ActorRef
 import akka.testkit.TestActor.AutoPilot
 import akka.testkit.{ TestActor, TestProbe }
-import com.google.inject.Provider
 import mesosphere.AkkaUnitTest
 import mesosphere.marathon.core.matcher.base.OfferMatcher.MatchedInstanceOps
 import mesosphere.marathon.test.MarathonTestHelper
 import mesosphere.mesos.protos.Implicits._
 import mesosphere.mesos.protos.OfferID
-import org.apache.mesos.Protos.DomainInfo.FaultDomain
-import org.apache.mesos.Protos.DomainInfo.FaultDomain.{ RegionInfo, ZoneInfo }
-import org.apache.mesos.Protos.{ DomainInfo, FrameworkID, Offer, SlaveID, OfferID => MesosOfferIdProto }
 
 class ActorOfferMatcherTest extends AkkaUnitTest {
```

Comment:
I understand that we removed `isInterestedIn` method but we should still unit test the use cases below.

Reply:
Done.